### PR TITLE
Introduced space-infix-ops and space-before-function-paren

### PR DESCRIPTION
**Pull request by pleszkowicz**
More details: 
- https://eslint.org/docs/rules/space-infix-ops
- https://eslint.org/docs/rules/space-before-function-paren

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -42,8 +42,10 @@ module.exports = {
         'no-unused-vars': [ 'error', { 'vars': 'all', 'args': 'none' } ],
         'object-curly-spacing': [ 'error', 'always' ],
         'semi': [ 'error', 'never' ],
+        'space-infix-ops': ["error", { "int32Hint": false }],
```
Comment (synaptiko) [inline index.js]: Just a small note, we use `'` instead of `"` in this file and also there is a space before `[` or `]` :-)

Comment (pleszkowicz) [reply index.js]: RIgtht :) done.

Comment (synaptiko) [reply index.js]: And what about the spaces? :-D

Comment (pleszkowicz) [reply index.js]: Holy moly :D we should apply this linter for itself. Fixed

Comment (synaptiko) [reply index.js]: Yeah, I had exactly the same idea :-D "Inception style" :-)

**Comment by synaptiko** [comment]
And now we need to release a new version, use it in `bayzat-fe`, update `yarn.lock` and create another PR :-D

**Comment by pleszkowicz** [comment]
and fix errors :) will do

**Pull request merged**